README.md

```diff
@@ -15,6 +15,15 @@ If you are still on `v1` make sure to follow [migration guide](https://vue-query
 
 If you are already on `v2`, just swap `vue-query` for `@tanstack/vue-query` in both of your `package.json` and `import` statements. Everything should still work as before.
 
+steps to move to `@tanstack/vue-query` from `vue-query` v2
+
+1. delete your `node_modules`
```

Comment:
Unnecessary step, node\_modules will be modified correctly when running `npm install` after editing package.json